CONTRIBUTING.md

```diff
@@ -0,0 +1,85 @@
+# How to contribute
+
+Operator Controller is an Apache 2.0 licensed project and accepts contributions via GitHub pull requests (PRs).
+
+This document outlines some conventions on commit message formatting, contact points for developers, and other resources
+to help Operator Controller contributors.
+
+## Communication
+
+- Email: [operator-framework-olm-dev](mailto:operator-framework-olm-dev@googlegroups.com)
+- Slack: [#olm-dev](https://kubernetes.slack.com/archives/C0181L6JYQ2)
+- Google Group: [olm-gg](https://groups.google.com/g/operator-framework-olm-dev)
+- Weekly in Person Working Group Meeting: [olm-wg](https://github.com/operator-framework/community#operator-lifecycle-manager-working-group) 
+
+## Contribution flow
+
+Any new contribution should be accompanied by a new or existing issue. This issue can help track work, discuss the
```

Comment:
Although we mention trivial fixes don't need a separate issue; where's the dividing line? A PR itself could be a bug report and fix in one, and having to create an issue (which, unrelatedly, share the same number spaces as PRs) seems like duplication of effort.